corehq/apps/cloudcare/views.py

```diff
@@ -220,7 +221,26 @@ def form_context(request, domain, app_id, module_id, form_id):
 cloudcare_api = login_or_digest_ex(allow_cc_users=True)
 
 
+def get_cases_vary_on(request, domain):
+    return [
+        request.couch_user.get_id
+        if request.couch_user.is_commcare_user() else request.REQUEST.get('user_id', ''),
+        request.REQUEST.get('ids_only', 'false'),
+        request.REQUEST.get('case_id', ''),
+        request.REQUEST.get('footprint', 'false'),
+        request.REQUEST.get('include_children', 'false'),
+        request.REQUEST.get('closed', 'false'),
+        json.dumps(get_filters_from_request(request)),
+        domain,
+    ]
+
+
+def get_cases_skip_arg(request, domain):
+    return string_to_boolean(request.REQUEST.get('cache_bust', 'true'))
```

Comment:
yea i initially didn't think it'd be a problem to cache the full result, but i only tested the simple case. as in: touchforms makes the call to grab the ids, then makes the call to grab the cases.  seems safer to just cache the ids. also you're totally right about the `cache_bust` thing. when i first wrote this, it was caching by default and i needed a param to break the cache. will update